version-history.md

```diff
@@ -180,14 +180,14 @@ This version history has been driven by what we hear from our wonderful communit
 ## v1.8 (released 2019-05-14)
 
 - **Browsers and host OSs now trust DDEV sites over HTTPS**. You do have to take the one-time action to install the CA keys into your operating system and browsers: `mkcert -install`. If you use one of the package installation methods or the Windows installer, you should already have `mkcert`. Otherwise, see the [mkcert](https://github.com/FiloSottile/mkcert) page for installation options. (Even curl and operating system tools generally trust the `mkcert` certs, and curl within the container also trusts them.)
-- **Dynamic container updates**: If you need extra Debian packages in your `web` or `db` container (or need to make more sophisticated adjustments) you no longer have to wait for them, or install them every time you start a project. You can add [`webimage_extra_packages`](https://ddev.readthedocs.io/en/latest/users/extend/customizing-images/#adding-extra-debian-packages-with-webimage_extra_packages-and-dbimage_extra_packages) to your `config.yaml` or [build a free-form Dockerfile](https://ddev.readthedocs.io/en/latest/users/extend/customizing-images/#adding-extra-dockerfiles-for-webimage-and-dbimage) (`Dockerfile.example` provided in your `.ddev` folder)
-- **nginx configuration in the web container has been reorganized** and simplified. This mostly won’t matter to you unless you use custom nginx configuration, in which case you’ll want to upgrade your configuration (see below). [docs](https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#providing-custom-nginx-configuration)
+- **Dynamic container updates**: If you need extra Debian packages in your `web` or `db` container (or need to make more sophisticated adjustments) you no longer have to wait for them, or install them every time you start a project. You can add [`webimage_extra_packages`](https://ddev.readthedocs.io/en/stable/users/extend/customizing-images/#adding-extra-debian-packages-with-webimage_extra_packages-and-dbimage_extra_packages) to your `config.yaml` or [build a free-form Dockerfile](https://ddev.readthedocs.io/en/stable/users/extend/customizing-images/#adding-extra-dockerfiles-for-webimage-and-dbimage) (`Dockerfile.example` provided in your `.ddev` folder)
```

Comment:
IMO we never really need to touch the version-history.md. It's so little used. Sometimes it helps us to understand the history of the project, not much more than that. But no need to change this.

Reply:
OK, I will revert this.